test/verify/check-users

```diff
@@ -793,6 +807,125 @@ class TestAccounts(MachineCase):
         b.wait_not_in_text(".pf-c-label-group__list", "testgroup0")
         wait(lambda: "testgroup0" not in m.execute("groups admin"))
 
+    @todoPybridge()
```

Comment:
This is highly dubious -- why would/should it fail on the pybridge? See above. Indeed it [succeeded](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230124-181926-0f2cd015-fedora-37-pybridge/log.html#335-2) on the current run.

Reply:
Answer to self: On main, TestAccounts.testBasic fails because there is apparently some excessive cockpit.spawn()ing going on, and the bridge eventually crashes as it runs out of fds. There is some leak in the bridge, or some excessive process calling in the page, or possibly both. Could very well be that this PR fixes it en passant. Anyway, as this test actually succeeded, let's just drop the decorator?